upgrading/topics/rhsso/changes-76.adoc

```diff
@@ -0,0 +1,183 @@
+
+
+Components known to have breaking changes:
+
+- Alert
+
+  - `action` prop changed to `actionClose`
+
+- Expandable
+
+  - renamed to `ExpandableSection`
+
+- Title
+
+  - size attr now uses `TitleSizes`
+
+- DataListContent
+
+  - `noPadding` changed to `hasNoPadding`
+
+- Grid, Stack, Level, Gallery
+
+  - `gutter` attr changed to `hasGutter`
+
+- Modal
+
+  - sizing control changed from, e.g. `isLarge`, to use `ModalVariant`, e.g. `variant={ModalVariant.large}`
+
+- Select
+
+  - `ariaLabelTypeAhead` to `typeAheadAriaLabel`
+
+  - `isExpanded` to `isOpen`
+
+  - `ariaLabelledBy` to `aria-labelledby`
+
+- DataListContent
+
+  - `noPadding` to `hasNoPadding`
+
+=== Client Policies Migration : client-scopes
+
+If you used a policy including client-scopes condition and edited JSON document directly, you will need to change the "scope" field name in a JSON document to "scopes".
+
+=== Liquibase upgraded to version 4.6.2
+
+Liquibase was updated from version 3.5.5 to 4.6.2, which includes, among other things, several bug fixes, and a new way of registering custom extensions using `ServiceLoader`.
+
+Closely follow the <<_upgrading,Upgrading Guide>>, specifically of *backing up
+existing database before upgrade*. While we did our best to test the consequences of the Liquibase upgrade, some installations could be using specific setup unknown to us.
+
+=== Proxy environment variables
+
+{project_name} now respects the standard `HTTP_PROXY`, `HTTPS_PROXY` and `NO_PROXY` environment variables for outgoing
+HTTP requests. This change could lead to unexpected use of a proxy server if you have for example the `HTTP_PROXY` variable defined but have no
+explicit proxy mappings specified in your SPI configuration. To prevent {project_name} from using those environment variables,
+you can explicitly create a no proxy route for all requests as `.*;NO_PROXY`.
+
+For more details, see the link:{installguide_link}#_proxy_env_vars[related chapter in the {installguide_name}].
+
+=== Deprecated features in the {project_operator}
+
+With this release, we have deprecated `podDisruptionBudget` field in the Keycloak CR of the https://github.com/keycloak/keycloak-operator[legacy {project_operator}].
+This optional field will be ignored when the Operator is deployed on Kubernetes version 1.25 and higher.
+
+As a workaround, you can manually create the Pod Disruption Budget in your cluster, for example:
+```yaml
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  labels:
+    app: keycloak
+  name: keycloak
+spec:
+  maxUnavailable: 1
+  selector:
+    matchLabels:
+      component: keycloak
+```
+See also the https://kubernetes.io/docs/tasks/run-application/configure-pdb/[Kubernetes Documentation].
+
+=== Keycloak Operator examples including unsupported Metrics extension
+
+Previously, an unsupported metrics extension was added in the example for the creation of the Keycloak CR by the {project_operator}. This has been removed.
```

Comment:
I wouldn't mention this. This mainly concerned the upstream, as in downstream we don't have the affected examples.